common/src/main/java/org/opensearch/ml/common/transport/connector/MLCreateConnectorRequest.java

```diff
@@ -39,8 +40,25 @@ public MLCreateConnectorRequest(StreamInput in) throws IOException {
     @Override
     public ActionRequestValidationException validate() {
         ActionRequestValidationException exception = null;
+
         if (mlCreateConnectorInput == null) {
-            exception = addValidationError("ML Connector input can't be null", exception);
+            return addValidationError("ML Connector input can't be null", exception);
+        }
+
+        String modelName = mlCreateConnectorInput.getName();
+        String description = mlCreateConnectorInput.getDescription();
+
+        if (modelName != null && !isSafeText(modelName)) {
+            exception = addValidationError(
+                "Model connector name can only contain letters, digits, spaces, underscores (_), hyphens (-), and dots (.)",
```

Comment:
This error message is different with MLUpdateConnectorRequest.java one. Should we keep them consistent ?

```
"Model connector name can only contain letters, digits, spaces, underscores (_), hyphens (-), and dots (.). Max length: 1000 characters.",
```

Reply:
Yes, in the next revision I kept it consistent.